cypress/integration/spec.js

```diff
@@ -125,8 +208,5 @@ describe('Chart Explorer', function() {
 
     cy.get('[data-test=vega-embed]')
     cy.screenshot('page with viz after reload')
```

Comment:
I realize you may not have written this code, but do you feel you understand what the value of taking a screenshot here is?

also I am of the opinion that `page with viz after reload` is a horrible file name (as i am understanding it to be a filename based on this doc https://docs.cypress.io/api/commands/screenshot.html#Syntax), so at the very least i think we should change this to something more aligned with standard file naming structure (no spaces...).

Reply:
Hmm good point I haven't questioned the purpose of why it was there in the first place. 

I also didn't realize that was a filename, that is horrible LOL.

But yeah I don't see a point in keeping the screenshots, I think it can be removed.